base-compat-batteries/base-compat-batteries.cabal

```diff
@@ -118,6 +119,7 @@ library
       Data.STRef.Compat
       Data.String.Compat
       Data.Type.Coercion.Compat
+      Data.Type.Equality.Compat
```

Comment:
You've added `Data.Type.Equality.Compat` to the `.cabal` file here, although there doesn't appear to be a corresponding file. Did you forget to add it?

Reply:
There is, you commented on it?